pkg/interactive/helper.go

```diff
@@ -0,0 +1,100 @@
+package interactive
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/briandowns/spinner"
+	"github.com/openshift/rosa/pkg/aws"
+	"github.com/openshift/rosa/pkg/interactive/confirm"
+	"github.com/openshift/rosa/pkg/output"
+	"github.com/openshift/rosa/pkg/rosa"
+	"github.com/spf13/cobra"
+)
+
+func GetOidcConfigID(r *rosa.Runtime, cmd *cobra.Command) string {
+	oidcConfigs, err := r.OCMClient.ListOidcConfigs()
+	if err != nil {
+		r.Reporter.Warnf("There was a problem retrieving OIDC Configurations "+
+			"for your organization: %v", err)
+		return ""
+	}
+	if len(oidcConfigs) == 0 {
+		return ""
```

Comment:
Previously we gave the user the option to enter the ID, so now they can only choose from the list retrieved from the backend?

Reply:
If the listing is empty they don't have anything they can enter. However, they can still pass the ID directly through the arg if needed